### ISSUE TITLE
Organize Imports: If all imports are to be removed, removes none of them

### ISSUE THREAD

**Issue by joshtynjala** [BowlerHatLLC/vscode-nextgenas#182]

**Comment by neminovno** [comment]
It looks this is not fixed in v0.11 here is example code:
```
<?xml version="1.0" encoding="utf-8"?>
<s:WindowedApplication xmlns:fx="http://ns.adobe.com/mxml/2009"
   xmlns:s="library://ns.adobe.com/flex/spark">

	<fx:Script>
		<![CDATA[
			import flash.events.MouseEvent;
			
		]]>
	</fx:Script>

</s:WindowedApplication>
```
Preview:
![2018-04-16_20-10-07](https://user-images.githubusercontent.com/1468887/38827120-2f406750-41b2-11e8-8366-ef53e233bf17.gif)

**Comment by joshtynjala** [comment]
It definitely works in *.as* files. I'll try to find what's wrong in *.mxml* files.
